packages/circuit-ui/components/Step/Step.spec.tsx

```diff
@@ -26,7 +26,9 @@ describe('Step', () => {
   });
 
   it('should throw error without children as a function', () => {
-    const expectedError = new Error('Children must be a function');
+    const expectedError = new Error(
+      '[Step] The `children` prop must be a function.',
+    );
```

Comment:
Exactly.

Reply:
I think we could do it, yes. Since we're almost fully TS now, we'll get errors if we pass the wrong props anyways. And then we can also cover this error logic if we'd like 🙂 

But we can also do this in a follow-up PR (I assume we'll need to fix a few tests), I don't mind the coverage drop

Reply:
I'm mostly worried about test failures in ze-dashboard.

Reply:
Good point 👀 They could probably also set the environment variable in their test env, but maybe we can give teams a bit more time until we turn on errors in specs